src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import duke.command.Command;
+
+import duke.exception.DukeException;
+
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Main Duke class
+ */
 public class Duke {
+
+    private final Parser parser;
+    private final TaskList taskList;
+    private final Ui ui;
+    private final Storage storage;
+
+    /**
+     * Basic Constructor
+     *
+     * @param filePath the path where file savedOutput.txt. which contains past state is located
+     * @throws DukeException
+     */
+    public Duke(String filePath) throws DukeException{
+        storage = new Storage(filePath);
+        ui = new Ui(48);
```

Comment:
Is there a reason for the ```Ui()``` constructor to need an argument of ```48```?

Reply:
Duke's output was wrapped around this speech bubble format. 48 was a default spacing size for the speech bubble. Any longer than 48 would require reformating and extension of the the speech bubble width. Should have made it into a static variable in UI itself to avoid confusion/add an explanation. Thanks for pointing it out!